src/features/orders/ordersSlice.ts

```diff
@@ -102,6 +102,28 @@ export const ordersSlice = createSlice({
 });
 
 export const { clear } = ordersSlice.actions;
-export const selectOrder = (state: RootState) => state.orders.orders[0];
+/**
```

Comment:
Yes - this was future proofing 😄 

Re: additional case where both orders are the same - happy to fall back to expiry. Will add this. One of the reasons I didn't was because it's actually not out of the question that the expiries are the same, given the resolution of the expiry time is one second (unix timestamp), and currently [MetaMask require a 150sec expiry](https://discord.com/channels/590643190281928738/831171587067347005/837311067423768656) so actually it's probably more likely that the expiry is the same than the token amount.

As you say though - might as well add it as it's relatively easy. I guess we don't need to worry about if expiry and amounts both match

Reply:
Nice! 🙂 thanks for the tidbit about the MetaMask expiry, wasn't aware of that.